src/simulation/shape_renderable.cpp

```
@@ -123,10 +123,10 @@ void ShapeRenderable::buildFromXML(const pugi::xml_node& item) {
       }
 
       int base_center_idx = (int)vertices.size() / 3;
-      int top_center_idx = base_center_idx + n_sectors + 2;
+      int top_center_idx  = base_center_idx + n_sectors + 2;
```

Clang (14.0.6)

[error] narrowing conversion from 'float' to 'int'

(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions,-warnings-as-errors)